jupyterhub/values.yaml

```diff
@@ -182,7 +182,8 @@ singleuser:
                 - 169.254.169.254/32
   events: true
   extraAnnotations: {}
-  extraLabels: {}
+  extraLabels:
+    hub.jupyter.org/network-access-hub: "true"
```

Comment:
Should we append this in `jupyterhub_config.py` instead perhaps? If not, utilizing the `extraLabels` option in the helm chart as a user would break the network policy function.

Reply:
oh, i thought the value files are merged recursively? so unless the user specifies:

```
singleuser:
  extraLabels: {}
```

or

```
singleuser:
  extraLabels:
    hub.jupyter.org/network-access-hub: null
```

the label should always be there.  but correct me if i am wrong.

Reply:
Ah yes you are right!

Reply:
But the first example would also work i think, then singleuser.extraLabels, being an empty dict, would merge with a populated dict.